runtime/include/blueoil_image.hpp

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2018 Leapmind Inc. */
+#ifndef RUNTIME_INCLUDE_BLUEOIL_IMAGE_HPP_
+#define RUNTIME_INCLUDE_BLUEOIL_IMAGE_HPP_
+
+#include "blueoil.hpp"
+
+namespace blueoil {
+namespace image {
+
+float *Tensor_at(Tensor &tensor, const int x, const int y); // return RGB
```

Comment:
Why it is not part of Tensor class itself?

Reply:
This "at" is a function for an image, as x, y args indicates.
If we include it in the Tensor class itself , It well be difficult interface a bit.

Reply:
So it is basically helper function. Wonder if here is better name, as `Tensor_at` sounds very generic. While the function expects a tensor of very concrete shape... :thinking: